packages/components/package.json

```
@@ -61,6 +61,7 @@
     "jest": "27.5.1",
     "jest-cli": "27.5.1",
     "npm-run-all": "4.1.5",
+    "puppeteer": "^21.5.2",
```

```suggestion
    "puppeteer": "21.5.2",
```

Need to re-generate the package-lock if you accept this change.